crates/aggchain-proof-builder/src/lib.rs

```diff
@@ -61,15 +61,28 @@ pub(crate) type ProverService = Buffer<
 /// proof generation. Collected from various sources.
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct AggchainProverInputs {
+    pub output_root: Digest,
```

Comment:
Some comment explanation about the output_root origin/value meaning would be nice.

Reply:
Added as part of the doc comment for `OutputRoot`